source/index.rst

```diff
@@ -43,20 +43,22 @@
    :columns: 3
 
    - :doc:`/cps/index`
+   - :doc:`/gmt/index`
    - :doc:`/obspy/index`
    - :doc:`/sac/index`
 
 .. rubric:: 绘图：
 
 .. hlist::
-   :columns: 1
+   :columns: 3
 
    - :doc:`/gmt/index`
+   - :doc:`/sac/index`
```

Comment:
SAC can be used for plotting.

Reply:
SAC102 can also be used to download data.